cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -28,8 +27,8 @@ def shuffle(l):
 
 
 @pytest.fixture(scope="function")
-def report_menus(group):
-    report_menus = menus.ReportMenu()
+def report_menus(group, appliance):
+    report_menus = appliance.collections.intel_report_menus.ReportMenu()
```

Comment:
Here we need to use the instantiate method, so that the class constructor (from attr) will get all of the attributes it needs - like appliance.

BaseEntity or BaseCollection derived classes should pretty much never be instantiated directly.